cfy_manager/components/globals.py

```diff
@@ -101,6 +104,22 @@ def _set_hostname():
         config[MANAGER][HOSTNAME] = socket.gethostname()
 
 
+def _apply_forced_settings():
+    if (
+        (
+            MANAGER_SERVICE not in config[SERVICES_TO_INSTALL]
+            and DATABASE_SERVICE in config[SERVICES_TO_INSTALL]
+        )
+        or (
+            DATABASE_SERVICE not in config[SERVICES_TO_INSTALL]
+            and MANAGER_SERVICE in config[SERVICES_TO_INSTALL]
+        )
+    ):
+        config[POSTGRESQL_SERVER][SSL_ENABLED] = True
+        config[POSTGRESQL_SERVER][ENABLE_REMOTE_CONNECTIONS] = True
+        config[POSTGRESQL_CLIENT][SSL_ENABLED] = True
```

Comment:
We could eliminate two of these by making TLS be always-on on the DB.

Reply:
And maybe we could check with Jonathan/Isaac whether ENABLE_REMOTE_CONNECTIONS needs to be kept- AFAIK the only use for that is if you want an AIO manager to allow external DB conns?

Reply:
i'm only moving this function, so i'm going to declare any actual TLS changes outside the scope of this code-level refactor

Reply:
Fair enough, will mark this thread as resolved.